azurerm/resource_arm_api_management_test.go

```diff
@@ -75,14 +75,15 @@ func TestAccAzureRMApiManagement_complete(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testCheckAzureRMApiManagementExists(resourceName),
 					resource.TestCheckResourceAttr(resourceName, "tags.Acceptance", "Test"),
+					resource.TestCheckResourceAttrSet(resourceName, "public_ip_addresses"),
```

Comment:
could we ensure there's items in this list? we can do that via 

```suggestion
					resource.TestCheckResourceAttrSet(resourceName, "public_ip_addresses.#"),
```